source/guide.md

```diff
@@ -0,0 +1,116 @@
+* Conflict --> these occur when more than one person edits a file, thus making 2 futures for the file that need to be reconciled into a single, new file; this can usually be fixed in text files with a merge
+* Merge --> resolving conflicts across branches or commits; also returning a branch to the Master
+* Push --> the act of sending your local committed changes to the remote version of that branch
+* Pull --> retrieving the remote version and making any required changes to your local repository
+* Sync --> a pull and then a push (retrieval and send)
+
+##### Intermediate
+
+* Stash --> storing any edits to files or deletions of files (but not the creation of new files) on a stack (stacks are first on, last off. This means that if you stash A, B, C. They will return to you as C, B, A)
+* Stash Pop --> retrieving the last stash off the stack
+* Staged --> changes you wish to commit or stash
+* Unstaged --> changes you do not wish to commit or stash but do not with to discard
+* Discard --> permanently throwing away a local change before commiting
+* Revert --> undoing a push by returning the remote repository to an earlier state
+
+### Suggested Workflows
+
+It will be up to each project to decide which workflow is best for it. Almost always, an agreed-upon structure on each team is critical, but the exact nature of that structure is up to the needs and proficiencies of those involved. I have worked on projects where we pulled hundreds of branches and reviewed people's code before it could merge, and I've worked on R&D projects where the only think I ever did was commit straight into the master without any branching at all. Each project is different, but here are some heuristics that have served me well:
+
+##### Branch and Merge, With Leads Committing to Master
+
+In this tactic, whenever anyone wants to do anything: add a feature, edit some code, upload art or music, delete a design document --they should pull a branch, commit changes into that branch, and then when they're ready to join everything back in, they should update that branch from the master branch and then submit a pull request (merge). Team leads, where appropriate, or the director, should then approve these. Usually these people can also commit straight into master, without cutting a branch, which will save the team time.
+
+##### Code Review
+
+Code Reviewing is the practice of having leads, usually engineers, look over peoples' code and suggesting fixes, changes, and bringing up possible bugs, edge cases, etc... that the original programmer may not have considered. Code Reviewers are usually skilled and experienced, but it may also be useful to have young programmers code review each others; even at a early stage, different programmers think differently and so this can be fruitful. The easiest way to code review is to look over commit history or check out pull requests (merges) from branches before approval.
+
+##### Pair Programming
```

Comment:
I kind of see how this relates to the heading directly above it, but I'm not sure this belongs in the Source Control article. The two above relate to VSC due to their integration with pull requests and branching, but pair programming seems to be a bit unrelated to the larger topic.